pkg/arvo/sys/arvo.hoon

```diff
@@ -44,6 +45,7 @@
   ==
 +$  cage  (cask vase)
 ++  cask  |$  [a]  (pair mark a)
++$  crag  (cask *)
```

Comment:
disagree re reusing `$card` fwiw, that has the connotation of being an event. we use `$mark` and `$desk` instead of `@tas`. but will remove `*`.

Reply:
OTOH maybe the desire here is that threads should take `$card`s?

Reply:
talked about this with @philipcmonk — we decided that `$card` is not a real `(cask)` and should instead be `(pair @tas *)` (since the type is not a mark). we also noticed that `$page` in clay is already what we want, and decided to promote it into arvo and use it instead of `$crag`.

Reply:
I disagree with this, at least in part. In the broader sense of `mark` as "symbolic content type", `card` is certainly a `(cask)`. Events, effects, userspace inputs (`%poke` payloads) and outputs (subscription updates), and successful scry responses all have the same shape -- a noun tagged with a symbolic name/description/type. The name used at all these different levels are by no means all backed by mark files, but that's already true of mark usage patterns in userspace.

I say "at least in part" because we might want to restrict `card` to mean "untyped event/effect" (although it already means more than that in userspace).

Reply:
Based on all this, I'm leaning:

- `$card` is a `(cask)` (I have no dog in the fight between it being that or `(pair @tas *)`).
- `$card` and `$page` are separate names. `$card` connotes an untyped arvo event, and `$page` connotes an untyped `$cage`.

Reply:
(or even `(pair event-type *)`)